test/suite/test_backup10.py

```diff
@@ -78,7 +85,10 @@ def test_backup10(self):
         orig_logs = [file for file in all_files if "WiredTigerLog" in file]
 
         # Now open a duplicate backup cursor.
-        config = 'target=("log:")'
+        if self.switch:
+            config = 'target=("log:switch")'
+        else:
+            config = 'target=("log:")'
         dupc = self.session.open_cursor(None, bkup_c, config)
```

Comment:
Should `dupc` get closed someplace?

Reply:
It is later in the test. This change only modified the config string of the existing test code.

Reply:
Hmm.  It looks like it gets closed on line 144, but that's included in the `if not self.switch:` block.  So it won't get closed in otherwise?

Reply:
Line 123. It is used in a couple error checks.

Reply:
Ahh.  Of course.  It was staring me in the face and I missed it.